src/forge/start.py

```diff
@@ -69,19 +67,22 @@ def start_fleet(n_list, config):
             client.start_instances(InstanceIds=[uid])
 
 
-def start(config):
+def start(config: Configuration):
     """start a stopped on-demand EC2 instance
 
     Parameters
     ----------
-    config : dict
+    config : Configuration
         Forge configuration data
     """
-    market = config.get('market')
+    market = config.market
 
     if 'spot' in market:
         logger.error('Master or worker is a spot instance; you cannot start a spot instance')
         # sys.exit(1)  # ToDo: Should we change the tests to reflect an exit or allow it to continue?
 
-    n_list = get_nlist({**config, 'rr_all': True})
+    start_config = config.clone()
```

Comment:
Do we need to clone the configs here?
Can we just set rr_all to True for the existing config?